src/Aspire.Hosting.Azure.EventHubs/PublicAPI.Unshipped.txt

```diff
@@ -5,5 +5,8 @@ Aspire.Hosting.Azure.AzureEventHubsResource.IsEmulator.get -> bool
 override Aspire.Hosting.Azure.AzureEventHubsEmulatorResource.Annotations.get -> Aspire.Hosting.ApplicationModel.ResourceAnnotationCollection!
 override Aspire.Hosting.Azure.AzureEventHubsEmulatorResource.Name.get -> string!
 static Aspire.Hosting.AzureEventHubsExtensions.RunAsEmulator(this Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsResource!>! builder, System.Action<Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsEmulatorResource!>!>? configureContainer = null) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsResource!>!
-static Aspire.Hosting.AzureEventHubsExtensions.WithDataBindMount(this Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsEmulatorResource!>! builder, string? path = null, bool isReadOnly = false) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.Azure.AzureEventHubsEmulatorResource!>!
```

Comment:
If we haven't shipped this API yet, I don't think we need to add all these Obsolete overloads.

Reply:
Forgot about that. I feel stupid as I reminded it to the EH team this morning.